src/main/groovy/io/seqera/wave/service/blob/impl/KubeTransferStrategy.groovy

```diff
@@ -61,18 +63,33 @@ class KubeTransferStrategy implements TransferStrategy {
 
     @Override
     BlobCacheInfo transfer(BlobCacheInfo info, List<String> command) {
-        final podName = podName(info)
-        final pod = k8sService.transferContainer(podName, blobConfig.s5Image, command, blobConfig)
-        final terminated = k8sService.waitPod(pod, blobConfig.transferTimeout.toMillis())
-        final stdout = k8sService.logsPod(podName)
-        final result = terminated
-                ? info.completed(terminated.exitCode, stdout)
+        final name = getJobName(info)
+        final job = k8sService.transferJob(name, blobConfig.s5Image, command, blobConfig)
+        final podList = k8sService.waitJob(job, blobConfig.transferTimeout.toMillis())
```

Comment:
Not sure how to handle this timeout. This is a problem similar to the build waiting vs running timeout

Reply:
I think, at timeout, we can check the status of job or pod and if its still running, increase the timeout otherwise kill it

Reply:
ok, i've doubled the total await time. See here 77808c04